inc/rain.tpl.class.php

```diff
@@ -478,7 +478,7 @@ protected function compileCode( $parsed_code ){
 				$value = "\$value$loop_level";           // value
 
 				//loop code
-				$compiled_code .=  "<?php $counter=-1; if( isset($var) && is_array($var) && sizeof($var) ) foreach( $var as $key => $value ){ $counter++; ?>";
+				$compiled_code .=  "<?php $counter=-1; if( $var !== null && is_array($var) && sizeof($var) ) foreach( $var as $key => $value ){ $counter++; ?>";
```

Comment:
Ok! Do you want me to update this PR or do I close it and you make the change on your side directly in raintpl?